src/chum.cpp

```diff
@@ -124,7 +125,7 @@ void Chum::refreshPackagesInstalled()
 {
   // check what repository provides installed packages
   QHash<QString,QString> packages;
-  for (const auto &p: m_packages_last_refresh_installed)
+  for (const auto &p: std::as_const(m_packages_last_refresh_installed))
```

Comment:
according to https://en.cppreference.com/w/cpp/utility/as_const, it is from C++17. maybe an issue for SFOS 3.1. In this context, I don't see much benefit of having it either

Reply:
Ok... qt uses qAsConst() macro for the same thing, but this was added in 5.7.

Reply:
I would suggest to skip it as we need to support older SFOS versions

Reply:
For future reference, ill link the check https://github.com/KDE/clazy/blob/master/docs/checks/README-range-loop-detach.md

Reply:
See failures at https://build.sailfishos.org/package/show/sailfishos:chum:testing/sailfishos-chum-gui . I have set revision to this branch commit